src/ansys/fluent/core/session_solver.py

```diff
@@ -352,35 +333,23 @@ def set_state(self, state: StateT | None = None, **kwargs):
     def __call__(self):
         return self.get_state()
 
-    def _populate_settings_api_root(self):
-        if not self._settings_api_root:
-            self._settings_api_root = _import_settings_root(self.settings)
-
-    def __getattr__(self, attr):
-        self._populate_settings_api_root()
-        if not attr.startswith("_") and attr in dir(self._settings_api_root):
-            if self.get_fluent_version() > FluentVersion.v242:
```

Comment:
I think this code was relevant for PyConsole 24.2. Fluent version check is not needed in current PyFluent.

Reply:
It was actually added so that it warns for only users of Fluent 25.1 +, but I think this is not relevant anyways, as this is a pyfluent change.